app/controllers/user_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UserController < ApplicationController
```

Comment:
Can you try removing this? I don't think that you need the UserController to make this work

Reply:
Done